lib/dns.js

```diff
@@ -272,12 +293,12 @@ exports.resolve = function(hostname, type_, callback_) {
   if (typeof resolver === 'function') {
     return resolver(hostname, callback);
   } else {
-    throw new Error('Unknown type "' + type_ + '"');
+    throw new Error(`Unknown type ${type_}`);
   }
 };
 
 
-exports.getServers = function() {
+exports.getServers = () => {
   return cares.getServers();
 };
```

Comment:
I think they're required in multi-line functions. I recall talking about it but I'm not sure what conclusion was reached and I'm fine with either - @thefourtheye ?